tests/Makefile

```diff
@@ -14,9 +14,9 @@ QGIS_VERSION_TAG:=3.16
 QGIS_MAP_WORKERS:=1
 WPS_NUM_WORKERS:=1
 
-LIZMAP_PORT:=127.0.0.1:9090
```

Comment:
No, interface must be explicit for security reason. If user want to change this he may create a docker-compose with its own options.

Reply:
The error message is not an issue ?

Reply:
The error message come from a bad local configuration.